src/components/Profile/ReturningModal/ReturningModal.js

```diff
@@ -0,0 +1,199 @@
+        let componentsList = [];
+
+        for (let i = 0; i < this.state.components.length; i++) { 
+            componentsList.push(
+                <Row className='sin_comp_backg_r container bottom-buffer'>
+                        <Col xs='6' className='container pad-left5'>
+                            <Col className='container'>
+                                { this.state.components[i].componentID }
+                            </Col>
+                        </Col>
+                        <Col xs='6' className='container'>
+                            <Col xs='2' className='col-pd hor-center align-items-center container'>
+                                <Button className='subt-button' 
+                                onClick={ () => this.handleDecrement(i) }
+                                >
+                                -
+                                </Button>
+                            </Col>
+                            <Col xs='4' className='item-counter col-pd ver-center hor-center container'>
+                                <div className="input-group-field">
+                                    { this.state.components[i].quantity }
+                                </div>
+                            </Col>
+                            <Col xs='2' className='col-pd hor-center align-items-center container'>
+                                <Button className='add-button' 
+                                onClick={ () => this.handleIcrement(i) }
+                                >
+                                +
+                                </Button>
+                            </Col>
+                            <Col xs='4' className='col-pd hor-center align-items-center container'>
+                                <Checkbox className='checkbox' />
+                            </Col>
+                        </Col>
+                </Row>
+            );
+        }
+        return componentsList;
+    }
+
+    /* In case that our user has no components or has not ever made a reservation, 
+    we have to throw a message, other case show reserved components table */
+    checkComponents() { 
+        if(this.usr_index === -1 || this.user_components.length === 0){
+            return (
+                <div>
+                    <h3> You have not active reserved components currently </h3>
+                </div>
+            )
+        }
+        else return (
+            <div>
+                <Row className="headers justify-content-center container bottom-buffer">
+                        <Col xs='6' className="align-left container pad-left5">
+                            <h5>Component</h5>
+                        </Col>
+                        <Col xs='6' className="align-left container pad-left5">
+                            <h5>Quantity</h5>
+                        </Col>
+                </Row>
+                { this.loadReserved() }
+            </div>
+        )
+    }
+
+
+    render(){
```

Comment:
Add space between function and bracket
```suggestion
    render() {
```